src/store/sparql.py

```diff
@@ -13,10 +17,31 @@ def setup(self):
     def run_sparql(self, query) -> QueryResult:
         """ Runs and returns the results from a sparql query"""
         self.sparql.setQuery(query)
-        self.sparql.setReturnFormat(JSON)
```

Comment:
did JSON not work? Always use a constant where one is available to you, strings can be typo'd, constants can't.